source/configuration/modules/mmrm1stspace.rst

```diff
@@ -0,0 +1,27 @@
+mmrm1stspace: First Space Modification Module
+=============================================
+
+**Author:** Pascal Withopf <pascalwithopf1@gmail.com>
+
+In rfc3164 the msg begins at the first letter after the tag. It is often the
+case that this is a unnecessary space. This module removes this first character
+if it is a space.
+
+Configuration Parameters
+------------------------
+
+Currently none.
+
+Examples
+--------
+
+This example receives messages over imtcp and modifies them, before sending
+them to a file.
+
+::
+   module(load="imtcp")
+   module(load="mmrm1stspace")
+   input(type="imtcp" port="13514")
+   action(type="mmrm1stspace")
+   action(type="omfile" template="outfmt" file="rsyslog.out.log")
```

Comment:
The template used here is not defined, so this does not work if someone tries it out. As the template itself is not important, I suggest to remove it. Also, a somewhat more common output file name may look nicer (e.g. "output.log") - but that's pesonal taste.